### PR TITLE
make compatible with Swift 4.2

### PR DESCRIPTION
**Pull request by sumit-anantwar**

### CODE REVIEW

File: Sources/ToastCenter.swift [hunk 1]
```diff
@@ -23,7 +23,7 @@ open class ToastCenter {
     #if swift(>=4.2)
     let name = UIDevice.orientationDidChangeNotification
     #else
-    let name = NSNotification.Name.UIDeviceOrientationDidChange
+    let name = UIDevice.orientationDidChangeNotification
```
Comment (Coeur) [inline Sources/ToastCenter.swift]: this change is incorrect, as we're in the #else for older Swift version: don't blindly trust the Migration assistant

File: Sources/ToastWindow.swift [hunk 2]
```diff
@@ -51,10 +51,10 @@ open class ToastWindow: UIWindow {
     let didBecomeActiveName = UIApplication.didBecomeActiveNotification
     #else
     self.windowLevel = .greatestFiniteMagnitude
-    let didBecomeVisibleName = NSNotification.Name.UIWindowDidBecomeVisible
-    let willChangeStatusBarOrientationName = NSNotification.Name.UIApplicationWillChangeStatusBarOrientation
-    let didChangeStatusBarOrientationName = NSNotification.Name.UIApplicationDidChangeStatusBarOrientation
-    let didBecomeActiveName = NSNotification.Name.UIApplicationDidBecomeActive
+    let didBecomeVisibleName = UIWindow.didBecomeVisibleNotification
```
Comment (Coeur) [inline Sources/ToastWindow.swift]: this change is incorrect, as we're in the #else for older Swift version: don't blindly trust the Migration assistant

**Comment by Coeur** [comment]
For the record, Swift 4.2 compatibility was already done on Master branch with #133.
The only needed changes from your pull request are in `.swift-version` and `.travis.yml`.

**Comment by devxoul** [comment]
@Coeur exactly. I'm preparing for the Swift 5 release today so I think this PR can be closed.

**Comment by Coeur** [comment]
The relevant changes were applied with #145, so I'm going to close that current PR.
If we miss anything, @sumit-anantwar, feel free to open a new PR, thank you.

**Pull request closed without merging**